config/manager/kustomization.yaml

```diff
@@ -5,4 +5,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: quay.io/openstack-k8s-operators/osp-director-operator
-￼ newTag: 0.0.1
+  newTag: 0.0.1
```

Comment:
there was some bad character and kustomize failed to set the tag when creating a new build